src/main/java/org/fundacionjala/coding/dennis/HighLow.java

```diff
@@ -0,0 +1,36 @@
+package org.fundacionjala.coding.dennis;
+
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.StringJoiner;
+
+/**
+ *
+ */
+public final class HighLow {
+
+    /**
+     *
+     */
+    private HighLow() {
+
+    }
+
+    /**
+     *
+     * @param numbers Cadena.
+     * @return cadena de 2 elementos.
+     */
+    public static String hal(final String numbers) {
+        StringJoiner stringResult = new StringJoiner(" ");
+        ArrayList<Integer> digitArray = new ArrayList<>();
+        for (String dString : numbers.split(" ")) {
+            digitArray.add(Integer.parseInt(dString));
+        }
+        Collections.sort(digitArray);
+        stringResult.add(Integer.toString(digitArray.get(digitArray.size() - 1)));
+        stringResult.add(Integer.toString(digitArray.get(0)));
+        return stringResult.toString();
```

Comment:
This may be refactored like this.
```java
        List<Integer> digitArray = new ArrayList<>();
        for (String dString : numbers.split(" ")) {
            digitArray.add(Integer.parseInt(dString));
        }
        digitArray.sort(Comparator.naturalOrder());
        return String.format("%d %d", digitArray.get(digitArray.size() - 1), digitArray.get(0));
```